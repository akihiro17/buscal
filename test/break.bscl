for i in 0 to 100 {
  if i == 2 {
    echo("break");
    break;
  }
  echo(format("i: {}", i));
}
