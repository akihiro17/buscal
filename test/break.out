i: 0
i: 1
break
